webapp/src/constants/errorMessages.js

```diff
@@ -1,8 +1,12 @@
-const errorMsg = {
+export const errorMsg = {
```

Comment:
These aren't localized, this file should probably go away. What's it used for?

Reply:
Good point, forgot that I initially created this but didn't remove when we were refactoring localization. I'll remove it and refactor